feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -235,20 +238,20 @@ func (tc *testCase) verifyInterfaceDUT(
 			}
 		}
 	}
-
-	disp := dip.Subinterface(0)
-
-	// IPv4 neighbor discovered by ARP.
-	dis4np := disp.Ipv4().Neighbor(atea.IPv4)
-	if got := dis4np.Origin().Get(t); got != dynamic {
-		t.Errorf("%s IPv4 neighbor %s origin got %v, want %v", dp, atea.IPv4, got, dynamic)
+	if !*deviations.MissingNeighborOrigin {
```

Comment:
If we are just missing the origin, can we still check the neighbor IP?

* [/interfaces/interface/subinterfaces/subinterface/ipv4/neighbors/neighbor/state/ip](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-interfaces.html#interfaces-interface-subinterfaces-subinterface-ipv4-neighbors-neighbor-state-ip)

* [/interfaces/interface/subinterfaces/subinterface/ipv6/neighbors/neighbor/state/ip](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-interfaces.html#interfaces-interface-subinterfaces-subinterface-ipv6-neighbors-neighbor-state-ip)

Reply:
Will check this and get back

Reply:
Is the entire neighbor subtree missing? Maybe the deviation should be renamed MissingNeighbor to better reflect the missing part?

Also during the investigation, I came to realize that BGP also has a neighbor subtree, so MissingSubinterfaceNeighbor is probably a better name in order to disambiguate.